README.md

```diff
@@ -28,3 +28,24 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
+
+### Run integration tests
+
+Requires `docker`, `cargo-hack` (installed via `cargo-install`).
```

Comment:
Let's be a bit more helpful here. Link to the [getting Docker guide](https://docs.docker.com/get-docker/), and give the actual instructions for installing `cargo-hack` along with a link to its GitHub.

Reply:
Done!